h/static/scripts/helpers/time-helpers.coffee

```diff
@@ -14,6 +14,7 @@ BREAKPOINTS = [
   [2 * day,    'yesterday',      day   ]
   [month,      '{} days ago',    day   ]
   [year,       '{} months ago',  month ]
+  [year*2,     'one year ago',   year  ]
```

Comment:
On the contrary, I'm so glad you pointed that out. I'm shocked that I didn't notice it myself because I'm usually so neurotic about formatting.